elyra/metadata/schemas/metadata-test.json

```
@@ -62,14 +62,21 @@
         "enum_test": {
           "description": "Property used to test properties with enums",
           "type": "string",
-          "enum": ["elyra", "rocks"]
+          "enum": ["elyra", "rocks"],
+          "uihints": {
+            "field_type": "dropdown",
+            "defaultChoices": ["elyra"]
```

let's apply the same convention here as well.
```suggestion
            "default_choices": ["elyra"]
```